aldryn_newsblog/models.py

```diff
@@ -235,7 +235,7 @@ def save(self, *args, **kwargs):
 
         # Ensure we aren't colliding with an existing slug *for this language*.
         if not Article.objects.translated(
-                slug=self.slug).exclude(pk=self.pk).exists():
+                slug=force_unicode(self.slug)).exclude(pk=self.pk).exists():
```

Comment:
This needs to be force_text() but I don't think it should be here at all, TBH. NewsBlog when used normally should never have lazy objects here.